frontend/src/metabase-lib/types.ts

```diff
@@ -130,9 +134,12 @@ export type ColumnDisplayInfo = {
   displayName: string;
   longDisplayName: string;
 
-  isCalculated: boolean;
-  isFromJoin: boolean;
-  isImplicitlyJoinable: boolean;
+  semanticType: string | null;
+  effectiveType: string;
```

Comment:
AFAIK we shouldn't be using these types. It's very error prone unless you understand what you doing. As I said in another comment, we shouldn't be inspecting this.

Reply:
60b6021cb111a02a67040231afcae72bc2f8107